ui/src/app/pages/data/data-set/genomic-extraction-modal.tsx

```diff
@@ -55,14 +60,10 @@ export const GenomicExtractionModal = ({
     }
   }, [workspaceNamespace]);
 
-  const runningExtract = fp.flow(
-    fp.orderBy((extract: GenomicExtractionJob) => extract.submissionDate, 'desc'),
-    fp.find((extract: GenomicExtractionJob) => extract.status === TerraJobStatus.RUNNING)
-  )(extractsForDataset);
-  const succeededExtract = fp.flow(
-    fp.orderBy((extract: GenomicExtractionJob) => extract.completionTime, 'desc'),
-    fp.find((extract: GenomicExtractionJob) => extract.status === TerraJobStatus.SUCCEEDED)
-  )(extractsForDataset);
+  const mostRecentExtract = fp.head(sortedExtractsForDataset);
```

Comment:
nit: you can combine throw this into the `fp.flow()` above and eliminate a variable

Reply:
oh true